lib/user.rb

```diff
@@ -0,0 +1,19 @@
+class User
+
+  attr_reader :name, :email, :username, :id
+
+  def initialize(id, name, email, username)
+    @id = id
+    @name = name
+    @email = email
+    @username = username
+  end
+
+  def self.create(name, email, username, password)
+    results = DatabaseConnection.query("INSERT INTO users (name, username, email, password)
```

Comment:
At some point we're gonna want to hash this password before it goes into the database